st2client/st2client/formatters/table.py

```diff
@@ -59,10 +67,23 @@ def format(cls, entries, *args, **kwargs):
                 first_col_width = col_width
 
             widths = []
+            subtract = None
             for index in range(0, len(attributes)):
+                attribute_name = attributes[index]
+
                 if index == 0:
                     widths.append(first_col_width)
                 else:
+                    if attribute_name in COLORIZED_ATTRIBUTES:
```

Comment:
We need to do that since if we break / wrap line incorrectly (e.g. before / in the middle of escape character), things will be messed up.

Previously we were just lucky that we never experienced this issue since "status" attribute never contained too much text (well, if a user had a really narrow terminal window, they would still experience this issue).